util/regexp-assemble/data/942362.data

```diff
@@ -3,138 +3,161 @@
-^[\W\d]+\s*?alter\s*login\b
-^[\W\d]+\s*?alter\s*mask\b
-^[\W\d]+\s*?alter\s*master\s*key\b
-^[\W\d]+\s*?alter\s*materialized\b
-^[\W\d]+\s*?alter\s*message\s*type\b
-^[\W\d]+\s*?alter\s*method\b
-^[\W\d]+\s*?alter\s*module\b
-^[\W\d]+\s*?alter\s*nickname\b
-^[\W\d]+\s*?alter\s*operator\b
-^[\W\d]+\s*?alter\s*outline\b
-^[\W\d]+\s*?alter\s*package\b
-^[\W\d]+\s*?alter\s*partition\b
-^[\W\d]+\s*?alter\s*permission\b
-^[\W\d]+\s*?alter\s*procedure\b
-^[\W\d]+\s*?alter\s*profile\b
-^[\W\d]+\s*?alter\s*queue\b
-^[\W\d]+\s*?alter\s*remote\b
-^[\W\d]+\s*?alter\s*resource\b
-^[\W\d]+\s*?alter\s*role\b
-^[\W\d]+\s*?alter\s*rollback\b
-^[\W\d]+\s*?alter\s*route\b
-^[\W\d]+\s*?alter\s*schema\b
-^[\W\d]+\s*?alter\s*search\b
-^[\W\d]+\s*?alter\s*security\b
-^[\W\d]+\s*?alter\s*server\b
-^[\W\d]+\s*?alter\s*service\b
-^[\W\d]+\s*?alter\s*sequence\b
-^[\W\d]+\s*?alter\s*session\b
-^[\W\d]+\s*?alter\s*symmetric\s*key\b
-^[\W\d]+\s*?alter\s*synonym\b
-^[\W\d]+\s*?alter\s*stogroup\b
-^[\W\d]+\s*?alter\s*table\b
-^[\W\d]+\s*?alter\s*tablespace\b
-^[\W\d]+\s*?alter\s*text\b
-^[\W\d]+\s*?alter\s*threshold\b
-^[\W\d]+\s*?alter\s*trigger\b
-^[\W\d]+\s*?alter\s*trusted\b
-^[\W\d]+\s*?alter\s*type\b
-^[\W\d]+\s*?alter\s*usage\b
-^[\W\d]+\s*?alter\s*user\b
-^[\W\d]+\s*?alter\s*view\b
-^[\W\d]+\s*?alter\s*work\b
-^[\W\d]+\s*?alter\s*workload\b
-^[\W\d]+\s*?alter\s*wrapper\b
-^[\W\d]+\s*?alter\s*xml\s*schema\b
-^[\W\d]+\s*?alter\s*xsrobject\b
-^[\W\d]+\s*?union\s*all\b
-^[\W\d]+\s*?union\s*select\b
-^[\W\d]+\s*?union\s*distinct\b
+
+    ##!> assemble
+{{nonword_number}}
+    ##!=>
+{{cmd_create}}\s+\w+
+{{cmd_delete}}\b
+{{cmd_desc}}\b
+{{cmd_drop}}\b
+{{cmd_insert}}\b
+{{cmd_load}}\b
+{{cmd_rename}}\b
+{{cmd_select}}\b
+{{cmd_truncate}}\b
+{{cmd_update}}\b
+      ##!> assemble
+{{cmd_alter_space}}
+      ##!=>
+aggregate\b
+application\s*role\b
+assembly\b
+asymmetric\s*key\b
+audit\b
+authorization\b
+availability\s*group\b
+broker\s*priority\b
+bufferpool\b
+certificate\b
+cluster\b
+collation\b
+column\b
+conversion\b
+credential\b
+cryptographic\s*provider\b
+database\b
+default\b
+dimension\b
+diskgroup\b
+domain\b
+endpoint\b
+extension\b
+external\b
+event\b
+flashback\b
+foreign\b
+fulltext\b
+function\b
+hierarchy\b
+group\b
+histogram\b
+index\b
+indextype\b
+inmemory\b
+instance\b
+java\b
+language\b
+large\b
+library\b
+lockdown\b
+logfile\s*group\b
+login\b
+mask\b
+master\s*key\b
+materialized\b
+message\s*type\b
+method\b
+module\b
+nickname\b
+operator\b
+outline\b
+package\b
+partition\b
+permission\b
+procedure\b
+profile\b
+queue\b
+remote\b
+resource\b
+role\b
+rollback\b
+route\b
+schema\b
+search\b
+security\b
+server\b
+service\b
+sequence\b
+session\b
+symmetric\s*key\b
+synonym\b
+stogroup\b
+table\b
+tablespace\b
+text\b
+threshold\b
+trigger\b
+trusted\b
+type\b
+usage\b
+user\b
+view\b
+work\b
+workload\b
+wrapper\b
+xml\s*schema\b
+xsrobject\b
+      ##!<
+union\s*all\b
+union\s*select\b
+union\s*distinct\b
```

Comment:
Move the `union` group up into the previous assemble block. This might change the generated expression but makes it much clearer what's happening.

Also add `cmd_union` for consistency.

Reply:
Using `{{cmd_union}}` is a good idea, I've changed. But when I move these lines into the block above, then `compare` shows some differences between the generated and existing expressions.

Reply:
Done.